packages/bpk-react-utils/package.json

```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "object-assign": "^4.1.1",
     "prop-types": "^15.5.8",
-    "react-transition-group": "^2.2.1",
-    "recompose": "^0.26.0"
+    "react-transition-group": "^2.4.0",
+    "recompose": "^0.27.1"
```

Comment:
I think this might break React 15 support https://github.com/acdlite/recompose/pull/647/files

Reply:
Seems like they're using https://github.com/reactjs/react-lifecycles-compat to keep it backwards compatible. I wonder if we should start doing the same for new components....